README.md

```diff
@@ -132,10 +132,73 @@ makepkg -si
 
 ## Usage
 
+> For a list of keyboard shortcuts press `?`
+
+Open the TUI with:
 ```bash
 $ lazysql
 ```
 
+### Connect to a DB
+
+1. start `lazysql`
+2. create a new connection (press `n`)
+3. provide a name for the connection as well as the URL to connect to (see <a href="#example-connection-urls">example connection URL</a>)
+4. connect to the DB (press `F3`)
+
+If you already have a connection set up:
+1. start `lazysql`
+2. select the right connection (press `j` and `h` for navigation)
+3. connect to the DB (press `c`)
+
+### Create a table
+
+There is currently no way to create a table from the TUI.
+However you can run the query to create the table as a SQL-Query, 
+inside the <a href="#execute-sql-querys">SQL Editor</a>.
+
+### Execute SQL querys
+
+1. Press `<Ctrl+E>` to open the built-in SQL Editor
+2. Write the SQL query
+3. Press `<Ctrl+R>` to execute the SQL query
+
+> To switch back to the table-tree press `H`
+
+### Open/view a table
+
+1. Expand the table-tree by pressing `e`
+2. Select the table you want to view
+3. Press `<Enter>` to open the table
+
+> To switch back to the table-tree press `H` \
+> To switch back to the table press `L`
+
+### Filter rows
+
+0. [Open a table](#openview-a-table)
+1. Press `/` to focus the filter input
+2. Write a `WHERE`-clause to filter the table
+3. Press `<Enter>` to submit your filter
+
+> To remove the filter, focuse the filter input (press `/`) and press `<Esc>`.
+
+### Insert a row
+
+0. [Open a table](#openview-a-table)
```

Comment:
Sometimes the lists starts with 1 and sometimes with 0, can we stick with one of the ways? maybe start with 1 always?